### PR TITLE
[deleted]

### PR DESCRIPTION
**Pull request by 1x6**

### CODE REVIEW

**Review by uhIgnacio-zz — CHANGES_REQUESTED** [review]
use the new headers/data otherwise spotify will be sus and prob will lock or ban the acc.

**Comment by 1x6** [comment]
didnt ask

**Comment by 1x6** [comment]
hm?

File: main.py [hunk 1]
```diff
@@ -39,18 +39,15 @@
     "referrer-policy": "strict-origin-when-cross-origin"
 })
 
-if "\"login_token\"" in resp:
-    os.system("cls" if os.name == "nt" else "clear")
+resp = requests.post('https://spclient.wg.spotify.com/signup/public/v1/account', data=data)
```
Comment (uhIgnacio-zz) [inline main.py:42]: data not defined.

**Review by uhIgnacio-zz — APPROVED** [review]
meh, ill merge it

**Pull request merged**